src/itp.rst

```diff
@@ -0,0 +1,218 @@
+we impose the implicit assumption that the theorem prover is error-free. In other words, we have to trust the theorem provers.
+
+
+Notable Computer-Aided Proofs
+=============================
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'leroy09'
+
+  The `CompCert <http://compcert.inria.fr/doc/index.html>`_ 
+  compiler generates assembly code from a large subset of C, and is written and checked in the interactive theorem prover `Coq <https://coq.inria.fr/documentation>`_.
+  
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'klein09'
+
+  seL4 is a formally verified microkernel. It provides high performance comparable to other L4 kernels, 
+  while it is also guaranteed never to crush,
+  never to perform an unsafe operation, and its behavior is completely predictable in every situation.
+
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'gonthier08'
+
+  The famouse four color theorem is proved with the aid of a computer at first,
+  and completely formalized in Coq later on.
+
+
+Gamification
+============
+There are several interesting works for gamification, one of them is `Polymorphic Blocks <https://cseweb.ucsd.edu/~lerner/pb.html>`_ .
+
+.. container:: bib-item
+
+  .. bibliography:: itp.bib
+    :filter: key == 'lerner15'
+
+  Polymorphic Blocks is purely graphical, and allows users to 
+  prove simple formulas in propositional logic. The idea is to use shapes to represent boolean connectives and
+  variables, blocks to represent inference rules and theorems, and composition of blocks to represent formal deductions.
+
+  In their evaluation, they recruited about 30 high school students who have no experience with logic, and randomly divided into two
+  groups; in the symbol group, students are required to do formal proofs about propositional logic, while in the game group, they are
+  required to do the exact same problem sets in the Polymorphic Blocks game. The average completion time of the game group is about half
+  of the symbolic group. After that, each group is given the same additional problem set, and participants are asked to work on that for
+  at least 5 minutes. They observed that in the game group, everyone finished the problems, while in the symbol group, most of them gave
+  up in the middle.
+
+  This paper concludes that the puzzle game like tool helps users to do proof much faster than a pen-and-paper interface.
+  It also results in more accuracy and engagement. The tool has a potential of teaching
+  logic, however the tool only teaches the user strategies of applying inference rules without going into details, and it is not clear
+  whether it helps the user understand the symbolic meaning of the formulas. The paper also states that this tool has a potential of
+  laying the foundation for crowdsourcing proofs via gamification.
+
+Proof General
```

Comment:
This isn't a general topic, it's a specific system. Maybe call these "Live Interactive Theorem Provers", and add the evaluation paper about KeY and Isabelle in to that section instead of having them separated